tests/network/connectivity/test_ovs_linux_bridge.py

```diff
@@ -29,6 +29,7 @@ class TestConnectivityLinuxBridge:
     )
     @pytest.mark.ipv4
     @pytest.mark.s390x
+    @pytest.mark.conformance
```

Comment:
The `L2_bridge_network` should fail in a single NIC cluster and/or without nmstate.
Adding the `multi_nic` marker should W/A that, no?